ovis_demo/include/ovis_demo.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef OVIS_DEMO_H
+#define OVIS_DEMO_H
+
+#include "Kinova.API.UsbCommandLayerUbuntu.h"
+#include "KinovaTypes.h"
+
+int (*initAPI)(void);
```

Comment:
Instead of adding these declarations, why not simply use the one provided in the header file ?